galleries/users_explain/text/text_intro.py

```diff
@@ -204,14 +202,14 @@
 # ======
 #
 # Subplot titles are set in much the same way as labels, but there is
-# the *loc* keyword arguments that can change the position and justification
-# from the default value of ``loc=center``.
+# the ``loc`` keyword argument that can change the position and justification
```

Comment:
We do have a convention to use _emphasis_ for arguments 
https://matplotlib.org/devdocs/devel/document.html#function-arguments

Reply:
Does this also apply to `loc="center"`?

Reply:
That is a relevant question for sure. I usually escape that by writing 
```
*loc* is ``'center'``
```

(This is the NumPy style, but that particular choice I do not fully agree on.)

Reply:
Please revert to
```suggestion
# the *loc* keyword argument that can change the position and justification
```

Background is that a parameter is more a specific name rather than a piece of code.

```
*loc* is “center“
```
and
```
``loc= “center“``
```
are equally ok. The former is regular text. The latter is specifying via a code expression. For simple cases, I‘d slightly prefer the text variant as is disrupts the flow of reading less.


@oscargus note that we do not put extra backticks around string values in text. https://matplotlib.org/devdocs/devel/document.html#quotes-for-strings

Reply:
Thanks, done!